### ISSUE TITLE
Module not found: Package path ./nocss is not exported from package [project]/node_modules/react-flow-renderer

### ISSUE THREAD

**Issue by neodon** [wbkd/react-flow#2082]
### Describe the Bug

I am trying to import the 'nocss' version of React Flow according to [Import Bundle without Styles](https://reactflow.dev/docs/guides/theming/#import-bundle-without-styles), and it is failing with an error message.

Here is the import statement I am using:

```typescript
import ReactFlow from 'react-flow-renderer/nocss'
```

And the error message when I try to build:

```
error - ./src/pages/index.tsx:3:0
Module not found: Package path ./nocss is not exported from package [project]/node_modules/react-flow-renderer
    (see exports field in [project]/node_modules/react-flow-renderer/package.json)
  1 | import * as React from 'react'
  2 | import type { NextPage } from 'next'
> 3 | import ReactFlow from 'react-flow-renderer/nocss'
```

I looked at `node_modules/react-flow-renderer/package.json` and the error message does seem to be correct.  There is no export defined for `nocss`:

```json
  "exports": {
    ".": {
      "import": "./dist/esm/index.js",
      "require": "./dist/umd/index.js"
    },
    "./package.json": "./package.json"
  }
```

This seems like it could be my build system or linter being a bit overzealous, but it is also a real issue.  I imagine the fix is trivial--just adding a value for the 'nocss' export in the exports field in react-flow-renderer's `package.json`.  Apologies if it is more complicated than that.

Here is my environment:
- Next.js 12.1.5
- NodeJS v17.9.0 (also tried v16.14.0 just in case)
- React Flow 10.1.0

Let me know if you would like any more information.

### Steps to Reproduce the Bug or Issue

1. Create a Next.js project with TypeScript -- `pnpm create next-app -- --ts`
2. Add `react-flow-renderer` as a dependency -- `pnpm install react-flow-renderer`
3. Import the 'nocss' version in a page component -- `import ReactFlow from 'react-flow-renderer/nocss'`
4. Attempt to start the app -- `pnpm dev`

### Expected behavior

I expect to see the 'nocss' version of React Flow rendered on the page.

### Platform

- OS: Windows 10.0.19044
- Browser: Chrome
- Version: 100.0.4896.127 (Official Build) (64-bit)

**Comment by moklick** [comment]
I missed the nocss build! Thanks for brining it up! I'll fix it asap.

**Comment by moklick** [comment]
Hey @neodon 

could you try `npm install react-flow-renderer@next` ?

**Comment by neodon** [comment]
Thanks for the fast response.  I can now successfully import `react-flow-renderer/nocss`.

I love this project.  It fills a niche I've been trying to find for years.  Keep up the good work!

**Comment by moklick** [comment]
fix released in v10.1.1
